.ci/container_setup.d/10-install-signing-service.sh

```diff
@@ -2,14 +2,25 @@
 
 set -eu
 
-if pulp --format none --refresh-api --base-url "http://localhost:8080" ${PULP_API_ROOT:+--api-root "${PULP_API_ROOT}"} --username "admin" --password "password" debug has-plugin --name "deb"
+has_deb=$(pulp --format none --refresh-api --base-url "http://localhost:8080" ${PULP_API_ROOT:+--api-root "${PULP_API_ROOT}"} --username "admin" --password "password" debug has-plugin --name "deb")
+has_ansible=$(pulp --format none --refresh-api --base-url "http://localhost:8080" ${PULP_API_ROOT:+--api-root "${PULP_API_ROOT}"} --username "admin" --password "password" debug has-plugin --name "ansible")
```

Comment:
Not sure this will work. You are interested in the exit code of these commands, not their stdout.

Reply:
You are correct, I removed the format option so now true or false is stored in the variables.